Xamarin.Forms.Platform.UAP/CellControl.cs

```diff
@@ -177,9 +177,28 @@ void OnLongTap(object sender, HoldingRoutedEventArgs e)
 				OpenContextMenu();
 		}
 
+		/// <summary>
+		/// To check the context, not just the text.
+		/// </summary>
+		MenuFlyout GetAttachedFlyout()
+		{
+			if (!(FlyoutBase.GetAttachedFlyout(CellContent) is MenuFlyout flyout))
```

Comment:
This is a really confusing way to declare flyout... I mean I get what it does but holy crap did it hurt my head for a second. Can you please flip this if statement around even though it will increase nesting.

Reply:
💯 really too confusing ...